packages/SwingSet/test/test-devices.js

```diff
@@ -16,6 +16,11 @@ function capargs(args, slots = []) {
   return capdata(JSON.stringify(args), slots);
 }
 
+test.before(async t => {
+  const kernelBundles = await buildKernelBundles();
+  t.context.data = { kernelBundles };
```

Comment:
Is `t.context` something provided by Ava or are we just doing this and hoping it will be safe?

Reply:
It's from AVA: https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#test-context